src/Billing/Controllers/StripeController.cs

```diff
@@ -750,8 +750,8 @@ private async Task<bool> AttemptToPayInvoiceWithAppleReceiptAsync(Invoice invoic
             GatewayType.AppStore, receiptLastTransactionId);
         if (existingTransaction != null)
         {
-            _logger.LogWarning("There is already an existing transaction for this Apple receipt.",
-                receiptLastTransactionId);
+            _logger.LogWarning(
+                $"There is already an existing transaction {receiptLastTransactionId} for this Apple receipt.");
```

Comment:
Don't use string interpolation in log messages. https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-library-authors#avoid-string-interpolation-in-logging but you should still have a hole for the value if we are sending one.

Reply:
Neat -- hadn't seen that personally. The Billing team can make the call on this one, but given it wasn't used it was presenting as a lint issue.

Reply:
Ideally using string interpolation there should be a linting error too. https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254

Along with these: 
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2253
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1727